aep/general/0193/aep.md.j2

```diff
@@ -1,5 +1,150 @@
+simplifies API client applications and eliminates the need for cumbersome
+custom error handling code.
+
+## Guidance
+
+Services **must** clearly distinguish successful responses from error
+responses.
+
+### Structure
+
+An error response **should** contain the following fields:
+
+| Name     | Type    | Required | Description                                                                                                                                                                                                                      |
+| -------- | ------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type     | string  | Y        | A unique identifier for the type of problem detected.                                                                                                                                                                            |
+| status   | integer |          | The HTTP status code that best describes the type of problem detected.                                                                                                                                                           |
+| title    | string  |          | A human-readable description of the problem. Messages are likely to be logged in plain text and should not include information about a specific occurrence. Information about specific occurrences should be part of `metadata`. |
+| detail   | string  |          | A human-readable explanation specific to this occurrence of the problem.                                                                                                                                                         |
+| instance | string  |          | A unique identifier for the specific occurrence of the problem.                                                                                                                                                                  |
+
+An error response may contain additional fields appropriate for a specific
+error type.
+
+The error title and detail fields **should** help a reasonably technical user
+_understand_ and _resolve_ the issue, and **should not** assume that the user
+is an expert in your particular API. Additionally, the error title and detail
+**must not** assume that the user will know anything about its underlying
+implementation.
+
+The error detail **should** be brief but actionable. Any extra information
+**should** be provided in additional properties. If even more information is
+necessary, you **should** provide a link where a reader can get more
+information or ask questions to help resolve the issue.
+
+The error detail field
+
+- is a developer-facing, human-readable "debug message"
+- is presented in the service's native language
```

Comment:
I get the intent behind this, but is a "service's native language" actually a well-defined concept?  Would it be clearer to say something like "all human-readable error details returned by an API must use the same language"?